pandas/io/excel/_base.py

```diff
@@ -1037,8 +1043,8 @@ def inspect_excel_format(
 
     Returns
     -------
-    str
-        Format of file.
+    str or None
+        Format of file (if it can be determined)
```

Comment:
nit: remove parentheses - the phrase is essential to the correctness of the statement and not supplemental. Also, period at the end.